action.js

```diff
@@ -196,4 +197,25 @@ module.exports = class {
       this.Jira.addApproversToIssue(this.issue.key, resp.data.values);
     }
   }
+
+  async autoAssignCreator() {
+    const { user } = this.githubEvent.pull_request;
+    if (!user) {
+      throw new Error(`No User on PR? - ${user}`);
+    }
+
+    const rigupUser = await this.dynamo.findByGithubId(user.id);
+
+    if (!rigupUser) {
+      throw new Error(`PR by unknown user? - ${user}`);
```

Comment:
What happens if we throw an error in a Github Action?

Reply:
Would it be better to log this instead of throwing an error?

Reply:
@himichaelroberts ah yes, lemme change to log

Reply:
@macleanevans it gets caught in index.js :)